src/language-js/index.js

```diff
@@ -6,64 +6,70 @@ const options = require("./options");
 const createLanguage = require("../utils/create-language");
 
 const languages = [
-  createLanguage(require("linguist-languages/data/javascript"), {
+  createLanguage(require("linguist-languages/data/JavaScript"), {
     override: {
       since: "0.0.0",
       parsers: ["babel", "flow"],
-      vscodeLanguageIds: ["javascript"]
-    },
-    extend: {
-      interpreters: ["nodejs"]
+      vscodeLanguageIds: ["javascript"],
+      interpreters: ["node", "nodejs"]
     }
   }),
-  createLanguage(require("linguist-languages/data/javascript"), {
+  createLanguage(require("linguist-languages/data/JavaScript"), {
     override: {
       name: "Flow",
       since: "0.0.0",
       parsers: ["babel", "flow"],
       vscodeLanguageIds: ["javascript"],
-
       aliases: [],
       filenames: [],
-      extensions: [".js.flow"]
+      extensions: [".js.flow"],
+      interpreters: ["node"]
```

Comment:
Don't think `node` can interpreted flow language

Reply:
not sure how this works, the original result has this, what is you suggestion? exclude `node` ?

Reply:
@fisker looks in original `linguist-languages` data and try to not lose `filenames` and `extensions`, other fields should be get from `linguist-languages`

Reply:
linguist-languages data for flow has `node` for interpreters, keep it?

Reply:
if `linguist-languages` has this field, don't put it in our code because it is already expanding

Reply:
this field is removed